docs/setup/installation.md

```diff
@@ -1,6 +1,6 @@
 ## How do I install it?
 
-For manual install instructions see the [README](https://github.com/dimagi/commcare-cloud/blob/master/README.md)
+For manual install instructions see [Manual Installation](./manual_installation.md)
```

Comment:
I'd like to describe who would benefit from manual vs automatic installation - any ideas?

This file describes one approach, and the "Manual Installation" file describes two others.

* installation.md describes setting up a user on a remote machine, pulling the repo, and running the init script
* manual install part 1 directs you to install some prerequisites, pull the repo, and run the script
* manual install part 2 directs you to make your own virtual env, pull the repo, install some dependencies, then use `manage-commcare-cloud` to finish setup.